### ISSUE TITLE
Install SSL Certificate

### ISSUE THREAD

**Issue by ahmedelfaleh** [mosip/mosip-infra#26]
Hello,

Where can I install the SSL certificate?

**Comment by ahmedelfaleh** [comment]
Please don't you gave any idea of what is happening?

**Comment by ahmedelfaleh** [comment]
Actually we were testing this platform to replace it with another one but no one replied to this issue.
it was under development at that time.
I noticed that they changed the whole architecture of the set up, did the issue occurred with you while you trying it with the new architecture?

**Comment by abdihakx** [comment]
Yes. I am running version 1.1.2 on-premise using self-signed certificates. I opened the issue here #667
